sources/models/defaults.toml

```diff
@@ -52,6 +52,24 @@ template-path = "/usr/share/templates/updog-toml"
 affected-services = ["updog"]
 seed.setting-generator = "bork seed"
 
+# Metrics
+
+[settings.metrics]
+# the URL to which anonymous health metrics will be sent
+metrics-url = "https://metrics.bottlerocket.aws"
+# whether or not health metrics will be sent. set to false to opt-out
+send-metrics = true
+# the list of services that are checked to determine if a host is healthy
```

Comment:
```suggestion
# the list of services that are checked to determine if a host is healthy, overridden in each variant to list services critical to that variant
```
I do like having a default here just in case, but I also think it'd be nice to clarify that we expect it to be overridden.

Reply:
Added in [b8b5c3a](https://github.com/bottlerocket-os/bottlerocket/compare/153acff977082318fef8fc8e61024b26574c362e..b8b5c3ad92298db2c46262eb2b6c5b6ea3f60187)